Sump/schema.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
please the definitions could be more precise (indeed some of the could be available already in the search database https://smartdatamodels.org/index.php/ddbb-of-properties-descriptions/
Review them